pkgs/by-name/sm/smartcat/package.nix

```
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+
+, openssl
+, pkg-config
+, xorg
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "smartcat";
+  version = "1.3.0";
+  src = fetchFromGitHub {
+    owner = "efugier";
+    repo = "smartcat";
+    rev = "main";
```

```suggestion
    rev = "refs/tags/v${version}";
```

always lock sources to a release tag (preferred) or commit ref